src/renderer/src/hooks/useAfterLogin.ts

```diff
@@ -366,6 +369,9 @@ const useFraudulentTokensNotification = () => {
                   to: 'account-tokens',
                   address: balance.address,
                   blockchain: balance.blockchain,
+                  queryParams: {
+                    hideFraudulentTokenHash: token.hash,
+                  },
```

Comment:
I think it would be easier remove the queryParams, as it does not need to be generic.

```suggestion
      address: balance.address,
      blockchain: balance.blockchain,
      hideFraudulentTokenHash: token.hash,
```